pkg/vm/cli/cli.go

```diff
@@ -13,6 +13,8 @@ import (
 	"strings"
 
 	"github.com/nspcc-dev/neo-go/pkg/compiler"
+	"github.com/nspcc-dev/neo-go/pkg/core"
+	"github.com/nspcc-dev/neo-go/pkg/core/interop"
```

Comment:
I don't like this dependency, `vm` (and `vm/cli`) is not supposed to depend on `core` in any way, maybe it's better to plug these interops in the CLI.

Reply:
What do you think about having a separate package with no dependencies where all names and convert-to-ID function are defined?

Reply:
I'm not sure it would help, you'll have to import packages implementing those functions there anyway to provide useful mapping.

Reply:
No, i am talking about
```
package names
const SystemContractCall = "System.Contract.Call"
func FromID(id uint32) string {
...
}
```
, which is imported in `vm` and maybe in `core`